llvm_util/utils.cpp

```diff
@@ -51,7 +53,7 @@ string value_name(const llvm::Value &v) {
   if (!v.getName().empty())
     return name = '%' + v.getName().str();
   return name = v.getType()->isVoidTy() ? "<void>"
-                                        : '%' + to_string(value_id_counter++);
+                                        : "%" + to_string(value_id_counter++);
```

Comment:
why this change?

Reply:
It was because visual studio code couldn't syntax highlight the following code if it was '%', just a silly reason :/ Undoed the change.